includes/rest-api.php

```diff
@@ -1,4 +1,10 @@
 <?php
+/**
+ * REST API
+ *
+ * @package    Secure Custom Fields
+ * @since      6.4.0
```

Comment:
Important note here: using 6.4.0 as we are not introducing the REST API in this PR, only fixing the missing file annotation that resulted in errors.

No action required!